tsconfig.json

```diff
@@ -2,18 +2,17 @@
   "compilerOptions": {
     "outDir": "./dist",
     "sourceMap": true,
-    "declaration": true,
-    "strictNullChecks": true,
-    "noImplicitAny": false,
+    "strict": true,
+    "jsx": "preserve",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "moduleResolution": "node",
-    "module": "CommonJS",
-    "target": "ES2016",
+    "module": "esnext",
+    "target": "esnext",
```

Comment:
The target should be targeting something that most browsers can still use. I think ES2016 is fine.

Reply:
Changed target back to ES2016